framework/transport/ieee1905_transport/ieee1905_transport_packet_processing.cpp

```diff
@@ -313,10 +296,12 @@ bool Ieee1905Transport::fragment_and_send_packet_to_network_interface(unsigned i
                 MAPF_WARN("bad packet format - extra bytes at end of message.");
                 return false;
             }
-            if (nextTlv->type() == 0 && remainingPacketLength > int(nextTlv->size())) {
-                MAPF_WARN("bad packet format - extra bytes after end of message TLV. "
-                          << remainingPacketLength);
-                return false;
+            if (nextTlv->type() == 0 &&  nextTlv->size() == 0) {
+                if (remainingPacketLength)
+                    MAPF_WARN("bad packet format - extra " << remainingPacketLength
```

Comment:
I don't know if this warning should even be there, because this *will* happen for a short relayed multicast message, e.g. topology notification.

Reply:
I'm not sure I understand - lets talk about this tomorrow.

Reply:
Easier to explain in writing :-)

The Ethernet minimum frame size is 60 octets. A topology notification CMDU is 14 (eth header) + 8 (CMDU header) + 9 (AL MAC address type TLV) + 3 (EOM TLV) = 34 octets. So, when the transport receives such a packet from the network, it will receive a 60-byte packet, not a 34-byte packet. remainingPacketLength will be 26.

Maybe we should add some infra to easily generate packets, so we can test stuff like this :-)

Reply:
Now I understand, thanks for that:)
And yes we should - good thing you got that python packet generator training now..

Reply:
My bad, this is on the *sender* side. For sending, there indeed should never be anything after the EOM TLV, so this warning should stay.